feature/stamps/src/main/java/io/github/droidkaigi/confsched2023/stamps/StampsScreen.kt

```diff
@@ -69,15 +85,123 @@ private fun StampsScreen(
         modifier = Modifier.testTag(StampsScreenTestTag),
         snackbarHost = { SnackbarHost(snackbarHostState) },
         content = { padding ->
-            Column(
-                Modifier
-                    .padding(padding),
+            val layoutDirection = LocalLayoutDirection.current
+
+            LazyColumn(
+                Modifier.padding(
+                    PaddingValues(
+                        top = padding.calculateTopPadding() + 20.dp,
+                        start = padding.calculateStartPadding(layoutDirection) + 16.dp,
+                        end = padding.calculateEndPadding(layoutDirection) + 16.dp
+                    )
+                ),
             ) {
-                Text(
-                    text = "Please implement StampsScreen!!!",
-                    style = MaterialTheme.typography.titleLarge,
-                )
+                item {
+                    StampsDetail()
+                }
+                item {
+                    Stamps(stamps = uiState.stamps, onStampsClick = onStampsClick)
+                }
```

Comment:
Instead of passing an array with `item`, how about using `items` and passing each element of the array individually? Specifically, I think it would look something like the following (though the naming 'Stamp' might not be correct).
```suggestion
                items(uiState.stamps) { stamp ->
                    Stamp(stamp = stamp, onStampsClick = onStampsClick)
                }
```
If you find it difficult to configure the layout using `LazyColumn`, there's also the option of using [LazyVerticalGrid](https://developer.android.com/jetpack/compose/lists?hl=ja#lazy-grids).

Reply:
@momomomo111 
Sure, I think it would be simpler to display each stamp using `items` for the stamp.

However, I don't think there is enough information for the `stamps` variable to represent the state (valid or invalid) of each stamp from A to E.

Specifically, the display order of the stamps and the state of each stamp.

Therefore, in the new commit([45cb057](https://github.com/DroidKaigi/conference-app-2023/pull/541/commits/45cb0573a66f21357824a3ec307af2bad915712d)), we changed the status of stamps from A to E as Boolean as shown below, displaying valid if true and invalid if false.
![image](https://github.com/DroidKaigi/conference-app-2023/assets/29651972/e9c7e3a1-3097-4339-a090-18ebe1d44b9d)

Regarding the display order, I felt that it was immutable, so I used Column and Row to display it.

I think that isStampA~E needs to acquire the state from local storage in StampsScreenViewModel and reflect the result, or change it according to the NFC processing result that will be implemented in the future.

I still don't quite understand the logic of this app.
So, if you can point out what I'm doing wrong, or what I should fix, I'd appreciate it.

Reply:
@5altNaCl 
Thank you for your suggestion!
If we set `isStampA` to `true`, we would need to re-compose not only the Compose function related to isStampA but also other places, which would not be efficient in terms of performance.
How about creating a `data class` and using `items` to reduce the coupling between stamps?
The `data class` of Sponsor might serve as a reference.

```kotlin
package io.github.droidkaigi.confsched2023.model

data class Stamp(
    val visibleImageResName: String,
    val hiddenImageResName: String,
    val isStampVisible: Boolean = false,
    val contentDescription: String
)
```
`example use for Android`
```kotlin
val stamp = Stamp("img_stamp_hoge_on", "img_stamp_hoge_off", true, "hoge")
val visibleImageResId = context.resources.getName(stamp.visibleImageResName)
```

`example use for iOS`
```swift
let stamp = KMMModule.Stamp(visibleImageResName: "img_stamp_hoge_on", hiddenImageResName: "img_stamp_hoge_off", isStampVisible: true, accessibilityDescription: "hoge")
let visibleImage = UIImage(named: stamp.visibleImageResName)
```

Thank you for bringing up the number of stamps!
Considering the potential increase in the number of stamps in the future, we should take that into account.
I apologize for the lack of clarity in the specifications. How about using [LazyVerticalGrid](https://developer.android.com/jetpack/compose/lists?hl=ja#lazy-grids) to display a maximum of two stamps side by side?
I assume you might be concerned about the display order, but it will be arranged according to the array's index, so it should be fine🙆‍♂️.

Reply:
I think we don't use Compose Multiplatform for stamp so we don't think we have to use res name 👀

Reply:
@5altNaCl 
The initial reason for proposing ResName was to accommodate KMM. However, it seems that this is not necessary, so there is no need to use ResName. I apologize for the confusion.🙏